File: Frontend/EasyCrudClient/src/app/models/programmer.ts
import { Availability } from './availability';
import { BankInfo } from './bankinfo';
import { BestTime } from './besttime';
import { Knoledge } from './knoledge';

export class Programmer {
  id: number = null;
  email: string = null;
  name: string = null;
  skype: string = null;
  phone: string = null;
  linkedin: string = null;
  city: string = null;
  state: string = null;
  portfolio: string = null;
  salary: number = null;
  bank: string = null;
  link: string = null;
  availability: Availability = null;
  bankInfo: BankInfo = null;
  bestTime: BestTime = null;
  knoledge: Knoledge = null;
}

File: Backend/EasyCrud/Models/Programmer.cs
namespace EasyCrud.Models
{
    public class Programmer
    {
        public long Id { get; set; }
        //Basic
        public string Email { get; set; }
        public string Name { get; set; }
        public string Skype { get; set; }
        public string Phone { get; set; }
        public string Linkedin { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Portfolio { get; set; }
        public int Salary { get; set; }
        public string Bank { get; set; }
        public string Link { get; set; }

        public virtual BankInfo BankInfo { get; set; }
        public virtual Knoledge Knoledge { get; set; }
        public virtual BestTime BestTime { get; set; }
        public virtual Availability Availability { get; set; }
    }
}
